File: php-sdk/examples/trending-news-watchlist.php
<?php

/*

	File: trending-news-watchlist.php
	Endpoint: public/trending-news-watchlist
	Repo: https://github.com/executium/trending-historical-cryptocurrency-news
	License: https://github.com/executium/trending-historical-cryptocurrency-news/LICENSE

*/

include '../src/Executium.trendingnews.Class.php';

# Declare your key and secret
$obj = new ExecutiumTrendingNews();

# Parameters required
$input=array();
$input['keyword_contains']='bitcoin';

# Send Request
$result = $obj->trending_news_watchlist($input);

# Result
echo '<pre>';
	echo "<strong>Sent</strong>\n\n";
	print_r($payload);
	echo "\n<strong>Returned</strong>\n\n";
	print_r($result);
echo '</pre>';

?>

File: python-sdk/examples/trending-news-watchlist.py

'''

	Endpoint: public/trending-news-watchlist
	Name: Trending News Watchlist

'''
import sys
import json

sys.path.insert(1, '../src')

# Include config and class
from config import key, secret
from Class import ApiAccess

# Declare your key and secret
obj = ApiAccess(key, secret)

# Declare endpoint
endpoint = 'public/trending-news-watchlist'

# No parameters required
payload = {}

result = obj.send(endpoint, payload, dict())

print(json.dumps(result['returned'], indent=4))